interface/portraits/portraits_highelf.gfx

```diff
@@ -274,21 +298,22 @@ spriteTypes = {
 		effectFile = "gfx/FX/portrait.lua"
 		layer = { # GFX_TYPE:[d|p]INDEX:COLOR_LINK:DONT_REFRESH_IF_VALID:CULTURE_INDEX
 			"GFX_character_background:p0"
-			"GFX_empty:p3"
-			"GFX_empty:p5"
-			"GFX_empty:p1:h:y"
+			"GFX_highelf_female_clothes_behind:p3"
+			"GFX_highelf_female_headgear_behind:p5"
+			"GFX_highelf_female_hair_behind:p1:h:y"
 			":p5:c6"
 
 			"GFX_highelf_female_base:p2"
 			"GFX_empty:d0"
-			"GFX_empty:p3"
+			"GFX_highelf_female_clothes:p3"
 			"GFX_empty:d1"
-			"GFX_empty:d2"
+			"GFX_highelf_female_cheeks:d4:h"
```

Comment:
Double cheeks.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/81196039-9b9d4900-8fcf-11ea-8a94-bbf1f41bff8e.png)

</details>